.github/workflows/go-verdiff.yaml

```diff
@@ -0,0 +1,24 @@
+name: go-verdiff
+on:
+  pull_request:
+    paths:
+    - go.mod
+    branches:
+    - main
+jobs:
+  go-verdiff:
+    runs-on: ubuntu-latest
```

Comment:
Is it is regular thing for the project i.e. ubantu-latest. ?

Reply:
Yes, it's based on other CI that we have.